### DISCUSSION TITLE
FlyPad V3 Push

### DISCUSSION THREAD

**Discussion by Blue2hark** [flybywiresim/aircraft#6977] [General]
Hello I find that the Push back of the new version and not top it was much better the Push of the previous version

**Comment by Benjozork** [comment]
We are aware and improvements will come

**Comment by andrejetcat** [comment]
I really love ALL what you and your team is doing here! Your work is so highly leveled and makes this Sim enjoyable! Without your work, MSFS would not be on my plate! - so please read this comment not as negative criticism:

BUT PLEASE bring back the "old" simple pushback feature. It was way more suitable for everyday use.
